### DISCUSSION TITLE
Access custom claims in jwt from policy condition

### DISCUSSION THREAD

**Discussion by anywhere88** [cerbos/cerbos#1673] [Q&A] [answered]
Hello,
we are trying to write conditions in yaml policies to access data from the jwt passed in the auxData. We could not find the right syntax to access data in custom claims.
We successfully access standard claims in this way: `request.aux_data.jwt.iss`
But we have a claim named "https://our-domain.com/claim/department", how do we access it? `request.aux_data.jwt.department` clearly doesn't work.

Thanks in advance.

**Comment by oguzhand95** [comment, marked as answer]
Hi @anywhere88,

Does this work?
```
request.aux_data.jwt["https://our-domain.com/claim/department"]
```

**Reply by anywhere88** [reply]
Hi @oguzhand95,
you are right, it works. I did even try it before, but I suppose Cerbos was not updated with the new policies. My bad.

Thanks a lot
